### ISSUE TITLE
Add KACARE stations

### ISSUE THREAD

**Issue by AdamRJensen** [AssessingSolar/solarstations#17]
SolarAnywhere benchmarking used a number of high-quality stations from KACARE (King Abdullah City for Atomic and Renewable Energy): https://www.solaranywhere.com/resources/data-validation/

![image](https://user-images.githubusercontent.com/39184289/160584976-701b3304-97a9-461e-a434-836d86bd7679.png)

**Comment by AdamRJensen** [comment]
Email sent to the Solar Energy authors on 30-01-2024 to enquire about which stations are still active.

The [API](https://www.energy.gov.sa:8443/index.html) provided only gives metadata.

**Comment by IoannisSifnaios** [comment]
More material [here](https://www.mdpi.com/2071-1050/10/4/1129) and the actual map of stations with photos is [here](https://rratlas.energy.gov.sa/RRMMDataPortal/MapTool).

Some info on instruments is [here](https://rratlas.energy.gov.sa/RRMMPublicPortal/sites/default/files/Solar%20Resource%20Data%20Documentation.htm).

The network looks active.

**Comment by AdamRJensen** [comment]
What is your "looks active" comment based on? Have you seen that you are able to get recent data or?

**Comment by IoannisSifnaios** [comment]
No, I did not get any data (it required to make an account, etc. so I did not look into that). I said that it "looks active" because they have grayed out some stations in their atlas, showing that they are not in operation. So I assumed that the rest are in operation... of course, that makes sense as long as they have an updated website (which I am not sure of)! I am also ok to change them to "?" if you think it makes more sense.
